### DISCUSSION TITLE
In-App Leverage Trading Module

### DISCUSSION THREAD

**Discussion by EmersonDickie** [block-wallet/extension#277] [Ideas]
People rely on centralized exchanges for doing their leverage trading as it offers efficiency. 

However, recently decentralized perpetual exchanges have took the spotlight. 

Block Wallet could offer an in-app module that offers leverage trading within the app. 

How this would work is Block Wallet leverages these perpetual exchanges on the backend. 

Similar to what Chainge Finance wallet is doing with GMX integration.

**Comment by zhenghao977** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
